Packet++/src/GvcpLayer.cpp

```diff
@@ -0,0 +1,143 @@
+#define LOG_MODULE PacketLogModuleGvcpLayer
+
+#include "GvcpLayer.h"
+#include <cstring>
+#include <sstream>
+
+namespace pcpp
+{
+	std::ostream& operator<<(std::ostream& os, GvcpCommand command)
+	{
+		os << "0x" << std::hex << static_cast<uint16_t>(command) << std::dec;
+		return os;
+	}
+
+	std::ostream& operator<<(std::ostream& os, GvcpResponseStatus status)
+	{
+		os << "0x" << std::hex << static_cast<uint16_t>(status) << std::dec;
+		return os;
+	}
+
+	/*---------------------- Class GvcpLayer ----------------------------*/
+
+	GvcpLayer::GvcpLayer(uint8_t* data, size_t dataSize, Layer* prevLayer, Packet* packet)
+	    : Layer(data, dataSize, prevLayer, packet, GVCP)
+	{}
+
+	GvcpLayer* GvcpLayer::parseGvcpLayer(uint8_t* data, size_t dataLen, Layer* prevLayer, Packet* packet)
+	{
```

Comment:
I think this method should also check if the data is not null and in the minimum size, for example `sizeof(GvcpRequestHeader)` if it's a request or `sizeof(GvcpAckHeader)` if it's a response, otherwise it should return a `PayloadLayer`